src/model.py

```diff
@@ -1106,15 +1108,15 @@ def init_components(self, subtasks):
                 "linear": torch.nn.Sequential(
                     torch.nn.Linear(
                         in_features=cnn_dim,
-                        out_features=1
+                        out_features=2
                     ),
                     torch.nn.Sigmoid()
                 ).to(torch.device(self.device))
             })
         })
 
         self.relu = torch.nn.ReLU()
-        self.scope_loss = torch.nn.BCELoss()
+        self.scope_loss = torch.nn.CrossEntropyLoss(ignore_index=self.ignore_id)
```

Comment:
Better loss for ignoring word pieces and paddings, preventing model confusing around what a scope is